doc-content/enterprise-only/processes/interacting-with-processes-viewing-process-instances-variables-advanced-filters-proc.adoc

```diff
@@ -1,7 +1,7 @@
-[id='interacting-with-processes-process-instances-special-filter-proc']
-= Defining special filters for process instances
+[id='interacting-with-processes-viewing-process-instances-variables-advanced-filters-proc']
+= Viewing process instance variables using advanced filters
 
-You can use the *Advanced Filters* option in {CENTRAL} to create special filters for process instances. When you create a special filter over the column `processId`, you can view the process instance variables of the process as columns using *Show/hide columns*.
+You can use the *Advanced Filters* option in {CENTRAL} to view process instance variables. When you create a filter over the column `processId`, you can view the process instance variables of the process as columns using *Show/hide columns*.
```

Comment:
Question - is "over the column" correct, maybe "for the"?

s/over the column `processId`/for the `processId` column